components/Footer.vue

```diff
@@ -116,29 +116,78 @@
           >Become a Sponsor</a>
         </p>
         <p class="copyright-text">
-          Organized and held by nwPlus Copyright © cmd-f 2020
+          Organized and held by nwPlus
+          <br>
+          Copyright © UBC Local Hack Day 2020
         </p>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-export default {
-  props: {
-    text: {
-      type: String,
-      default: 'Organized and held by nwPlus. Copyright © nwHacks 2019'
-    }
-  },
-  data() {
-    return {
-      splitText: this.text.split('.')
-    }
-  }
-}
+export default {}
```

Comment:
What is this exporting?

Reply:
Oh my bad, the stuff from before wasn't being used so I can just remove it.